composer.json

```diff
@@ -46,51 +31,68 @@
         "phpcr/phpcr-shell": "~1.0",
         "symfony/debug": "^3.4",
         "symfony/debug-bundle": "^3.4",
+        "symfony/dotenv": "^3.4",
         "symfony/inflector": "^3.4",
         "symfony/phpunit-bridge": "^3.4",
         "symfony/var-dumper": "^3.4",
         "symfony/web-profiler-bundle": "^3.4",
         "symfony/web-server-bundle": "^3.4"
     },
+    "autoload": {
+        "psr-4": {
+            "App\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "App\\Tests\\": "tests/"
+        }
+    },
+    "replace": {
+        "symfony/polyfill-ctype": "*",
+        "symfony/polyfill-iconv": "*",
+        "symfony/polyfill-php71": "*",
+        "symfony/polyfill-php70": "*",
+        "symfony/polyfill-php56": "*"
+    },
     "scripts": {
-        "post-install-cmd": [
-            "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::buildBootstrap",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::clearCache",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installAssets",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::installRequirementsFile",
-            "Sensio\\Bundle\\DistributionBundle\\Composer\\ScriptHandler::prepareDeploymentTarget",
+        "auto-scripts": [
+            "bin/adminconsole cache:clear",
+            "bin/websiteconsole cache:clear",
+            "bin/adminconsole assets:install --symlink --relative"
+        ],
+        "auto-sulu-scripts": [
```

Comment:
Any reson for splitting this up into two different auto scripts? They are always used together, aren't they?